.github/workflows/test.yml

```diff
@@ -1,72 +1,73 @@
-#  push:
-#    tags: ""
-#    branches: master
-#    paths-ignore:
-#      - "**/*.md"
-#      - "charts/**"
-#  # We also run tests on pull requests targeted at the master branch.
-#  pull_request:
-#    branches: master
-#    paths-ignore:
-#      - "**/*.md"
-#      - "charts/**"
-#
-#jobs:
-#  test:
-#    runs-on: ubuntu-20.04 # newest available distribution, aka focal
-#    if: "!contains(github.event.head_commit.message, 'maven-release-plugin')"
-#    steps:
-#      - name: Checkout Repository
-#        uses: actions/checkout@v2
-#        with:
-#          fetch-depth: 0 # full git history for license check
-#      - name: Cache local Maven repository
-#        uses: actions/cache@v2
-#        with:
-#          path: ~/.m2/repository
-#          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-#          restore-keys: ${{ runner.os }}-maven-
-#      - name: Cache NPM Packages
-#        uses: actions/cache@v2
-#        with:
-#          path: ~/.npm
-#          key: ${{ runner.os }}-npm-packages-${{ hashFiles('zipkin-lens/package-lock.json') }}
-#      - name: Test without Docker
-#        run: build-bin/maven_go_offline && build-bin/test -Ddocker.skip=true
-#  test_docker:
-#    runs-on: ubuntu-20.04 # newest available distribution, aka focal
-#    if: "!contains(github.event.head_commit.message, 'maven-release-plugin')"
-#    strategy:
-#      matrix:
-#        include:
-#          - name: zipkin-collector-kafka
-#          - name: zipkin-collector-rabbitmq
-#          - name: zipkin-storage-cassandra
-#          - name: zipkin-storage-elasticsearch
-#          - name: zipkin-storage-mysql-v1
-#    steps:
-#      - name: Checkout Repository
-#        uses: actions/checkout@v2
-#        with:
-#          fetch-depth: 1 # -Dlicense.skip=true so we don't need a full clone
-#      - name: Cache local Maven repository
-#        uses: actions/cache@v2
-#        with:
-#          path: ~/.m2/repository
-#          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-#          restore-keys: ${{ runner.os }}-maven-
-#      # We can't cache Docker without using buildx because GH actions restricts /var/lib/docker
-#      # That's ok because DOCKER_PARENT_IMAGE is always ghcr.io and local anyway.
-#      - name: Test with Docker
-#        run:
-#          | # configure_test seeds NPM cache, which isn't needed for these tests
-#          build-bin/maven/maven_go_offline &&
-#          build-bin/docker/configure_docker &&
-#          build-bin/test -pl :${{ matrix.name }} --am -Dlicense.skip=true
+---
+name: test
+
+# We don't test documentation-only commits.
+on:
+  # We run tests on non-tagged pushes to master that aren't a commit made by the release plugin
+  push:
+    tags: ""
+    branches: master, zipkin-v3
+    paths-ignore:
+      - "**/*.md"
+      - "charts/**"
+  # We also run tests on pull requests targeted at the master branch.
+  pull_request:
+    branches: master, zipkin-v3
```

Comment:
I am feeling this doesn't take effects? Need merging?

Reply:
Yes, I think it should be worked after merging.

Reply:
I feel you should submit this to the main branch to make it works.

Reply:
But it has different project names. If I added to the main branch, all new PRs to the main branch should fail.

Reply:
I mean only branch name for the GHA on condition, rather than other parts of changes.